node/clients/masterdata.ts

```diff
@@ -50,13 +50,18 @@ export class MasterData extends ExternalClient {
     acronym: string,
     fields: string[],
     where: string,
-    pagination: PaginationArgs
-  ) =>
-    this.get<T[]>(this.routes.search(acronym), {
+    pagination: PaginationArgs,
+    schema?: string
+  ) =>{
+    return this.get<T[]>(this.routes.search(acronym), {
       headers: paginationArgsToHeaders(pagination),
       metric: 'masterdata-searchDocuments',
-      params: { _fields: generateFieldsArg(fields), _where: where },
-    })
+      params: {
+        _fields: generateFieldsArg(fields),
+        _where: where,
+        ...schema? {_schema: schema} : null
```

Comment:
maybe missing a space ?
```suggestion
        ...schema ? {_schema: schema} : null
```